.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  extends: 'skyscanner',
+  env: {
+    node: true,
+    jest: true,
+  }
+};
```

Comment:
Everything else uses `.eslintrc` - this would be diverting from that standard (pretty ironic for a module which is aimed at standardising our CSS :P)